### PR TITLE
Add user model

### PR DESCRIPTION
**Pull request by wleoguty91**
## Summary

- Added user model with relations to books, also added the bookalreadyowned exception

## Trello Card

- https://trello.com/c/FaosxTHj/19-crear-modelo-de-user

### CODE REVIEW

File: src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package wolox.training.exceptions;
+
+public class BookAlreadyOwnedException extends Exception{
```
Comment (leonplondon) [inline src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java]: Fix identation.

Comment (wleoguty91) [reply src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java]: Done!

File: src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package wolox.training.exceptions;
+
+public class BookAlreadyOwnedException extends Exception {
+
+    public BookAlreadyOwnedException() {
```
Comment (facucompe) [inline src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java]: Do you need to write the constructor?

**Review by facucompe — DISMISSED** [review]

File: src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java (hunk 2 above)
Comment (leonplondon) [reply src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java]: Use RuntimeException and you avoid this empty constructor.

File: src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package wolox.training.exceptions;
+
+public class BookAlreadyOwnedException extends RuntimeException {
+
```
Comment (leonplondon) [inline src/main/java/wolox/training/exceptions/BookAlreadyOwnedException.java:4]: Nice Job!!!

**Review by leonplondon — APPROVED** [review]

**Pull request merged**